btr-web/btr-common-components/interfaces/canadian-citizenship-e.ts

```diff
@@ -0,0 +1,5 @@
+export enum CANADIAN_CITIZENSHIP_E {
```

Comment:
Can you put this in the enums folder? And for enums we've been doing camel case for the name so to keep it consistent this would be `export enum CanadianCitizenshipE`

Reply:
Removed the Enum completely. For some reason I could not make that stuff work. Vue components kept complaining.